pom.xml

```diff
@@ -1,582 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Why did you delete the pom file ??

Reply:
I deleted it by wrong sorry about this mistake, I have limited experience with Git.